paper.tex

```diff
@@ -48,7 +48,9 @@
 \begin{document}
 
 \section{Introduction}
-The growth of data marketplaces is an inevitable result of the IoT (Internet of Things) revolution. As physical assets such as ships, factories, vehicles, farms and buildings become digital, their digital twins will gradually act as secure data exchanges.\cite{digitaltwin, AutonomousDriving} As data streams surge across silos and carry value across organizations, traditional value chains will transition into a web of value. This paradigm shift will be more complex to administers, forcing businesses to rethink their competitive play as part of these ecosystems. Data marketplaces will emerge as a means to exchange data, monetize data streams and provide the basis of new business models. We refer to this new wave of value creation, for the Internet of Everything, as the "Economy of Things." There are three main barriers to achieving data marketplace:
+As the future that some indirect information such as web-surfing hours, location tracking data or even daily electric meter records can unveil an individual’s lifestyle comes, an insurance company may easily compute customized premiums while marketing industry can target potential consumers with precise personal advertisement. Inevitably, the value of data is explicitly identified and connected to current business model because of the spread of personal devices, data produced by end-users has faced with new challenges such as ownership fights, reasonable use and not surprisingly -- trading spaces with lawful basis.
```

Comment:
Regarding web-surfing, shall we cite [basic attention token](https://basicattentiontoken.org/) and take it as an example?